packages/edit-site/src/store/private-selectors.js

```diff
@@ -19,3 +26,8 @@ export function getCanvasMode( state ) {
 export function getEditorCanvasContainerView( state ) {
 	return state.editorCanvasContainerView;
 }
+
+export function isEntityFocusMode( state ) {
```

Comment:
`isEntityFocusMode` sounds a big confusing to me. Why not name it like what it does, checking focusable entities, so more like `isEntityFocusable`?